controllers/SitemapController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * Pimcoresitemapplugin_SitemapController
+ *
+ * @author Yann Weyer
+ */
+
+use Byng\Pimcore\Sitemap\SitemapPlugin as SitemapPlugin;
+
+class Pimcoresitemapplugin_SitemapController extends \Pimcore\Controller\Action\Frontend
+{
+
+    /*
```

Comment:
I think PhpDoc requires `/**` to be the start of a doc block. Could `@return void` also be added here too please?